iOS-Weather-App/AppDelegate.swift

```diff
@@ -10,27 +10,45 @@ import UIKit
 @main
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
-
+    var window: UIWindow?
+    private var navigationController: UINavigationController?
```

Comment:
hold only navigationController, other properties should be global at this class

Reply:
global as static or public?